src/Draw.php

```diff
@@ -6,7 +6,7 @@
 
 /**
  *  Draw - class extension with drawing methods
- *
+ *©
```

Comment:
:D

Reply:
Yup, as soon as my push went up I saw it. Nothing is ever easy, doubly so when your own little scripts start giving you attitudes. I'm glad this ended up in comments. Now I need to scour the rest of my code from the last week to see where else this has been injected.